cmd/river/go.mod

```diff
@@ -2,7 +2,7 @@ module github.com/riverqueue/river/cmd/river
 
 go 1.21.4
 
-// replace github.com/riverqueue/river => ../..
+replace github.com/riverqueue/river => ../..
```

Comment:
Uncommented to demonstrate that the whole package works, but I'll have to unwind this as I'm cutting a release. We have steps in `development.md` that'll theoretically work for this situation ...